aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -166,8 +150,7 @@ def get_query_string_value(self, name: str, default_value: Optional[str] = None)
         str, optional
             Query string parameter value
         """
-        params = self.query_string_parameters
-        return default_value if params is None else params.get(name, default_value)
+        return get_query_string_value(self.query_string_parameters, name, default_value)
```

Comment:
prefer kwargs to ease reading/refactoring signature later

Reply:
Make total sense, Heitor!
Fixed